java/runtime/src/main/java/org/ray/runtime/raylet/MockRayletClient.java

```diff
@@ -41,7 +46,9 @@ public void onObjectPut(UniqueId id) {
   public void submitTask(TaskSpec task) {
     UniqueId id = isTaskReady(task);
```

Comment:
in `isTaskReady`, except checking arguments, we should also check `task.getDependencies()`. Note: dependencies now only contain actor dummy objects. So, you should also mark the dummy object as locally available when an actor task finishes. dummy object is the last return value of an actor task.